build action to push to NuGet

wolf-361 opened the pull request:
CI/CD to build and push to NuGet

Could make it conditional to the versionning changing ?

Hillpro commented:
Shouldn't subscribe to the pull_request event, since every newly created PR will trigger a publish, even if not merged/closed.


> Could make it conditional to the versionning changing ?

I don't think this would be necessary, since every PR should increase the version number. Every merge into the master branch should already be production ready.

Hillpro commented:
.NET Setup is done with dotnet/Sdk/6.0.417, but the .NET pack is done with dotnet/sdk/7.0.403. Might be the error

wolf-361 commented:
Fixed the dotnet issue, the problem was that dotnet 7 was already installed and we were not specifying to use dotnet 6 in the build (this is done with a global.json file)

I also removed the on PR action, it was an error, thanks !
